python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -702,13 +702,9 @@ def test_is_supported(arg, supported):
 
 def test_groupby_unique_lists():
     df = pd.DataFrame({"a": [0, 0, 0, 1, 1, 1], "b": [10, 10, 10, 7, 8, 9]})
-    ddf = dd.from_pandas(df, 2)
     gdf = cudf.from_pandas(df)
     gddf = dask_cudf.from_cudf(gdf, 2)
-    dd.assert_eq(
-        ddf.groupby("a").b.unique().compute(),
-        gddf.groupby("a").b.unique().compute(),
-    )
```

Comment:
Does this no longer pass?

Reply:
Yes, we fixed a bug that `dask` is yet to fix:
```
In [6]: df = pd.DataFrame({"a": [0, 0, 0, 1, 1, 1], "b": [10, 10, 10, 7, 8, 9]})

In [7]: df.groupby("a").b.unique()
Out[7]: 
a
0         [10]
1    [7, 8, 9]
Name: b, dtype: object

In [8]: import dask.dataframe as dd

In [9]: ddf = dd.from_pandas(df, 2)

In [10]: ddf.groupby("a").b.unique().compute()
Out[10]: 
a
0       [10]
1    [7 8 9]
Name: b, dtype: string

In [11]: ddf.groupby("a").b.unique().compute()[1]
Out[11]: '[7 8 9]' # Wrong, Should be a list of 3 numbers `[7, 8, 9]` than a raw string.
```

Reply:
Ah okay, thanks for the info. Looks like it works with the dask-expr backend, but this change makes sense to me. Thanks!